core/src/main/java/io/questdb/cairo/MetadataCache.java

```diff
@@ -186,6 +186,7 @@ private void hydrateTable0(@NotNull TableToken token, boolean throwError) {
             table.setMaxUncommittedRows(metaMem.getInt(TableUtils.META_OFFSET_MAX_UNCOMMITTED_ROWS));
             table.setO3MaxLag(metaMem.getLong(TableUtils.META_OFFSET_O3_MAX_LAG));
             table.setTimestampIndex(metaMem.getInt(TableUtils.META_OFFSET_TIMESTAMP_INDEX));
+            table.setTtlHoursOrMonths(metaMem.getInt(TableUtils.META_OFFSET_TTL_HOURS));
```

Comment:
nit: table utils constant name should be consistent with the meaning of the value